modules/widget-visibility/widget-conditions/widget-conditions.js

```diff
@@ -200,32 +200,36 @@ jQuery( function( $ ) {
 			select = condition.find( '.conditions-rule-minor' ).html( '' ),
 			major = condition.data( 'rule-major' );
 
-		if ( ! major ) {
+		if ( ! major || major === 'post_type' ) {
 			select.attr( 'disabled', 'disabled' );
 			return;
 		}
 
 		minor = condition.data( 'rule-minor' );
 		hasChildren = condition.data( 'rule-has-children' );
-		majorData = widget_conditions_data[ major ];
 
-		for ( i = 0, _len = majorData.length; i < _len; i++ ) {
-			key = majorData[i][0];
-			val = majorData[i][1];
+		if ( widget_conditions_data.hasOwnProperty(major) ) {
```

Comment:
I'd say that the change below is not needed because this can only be false when `major` is `post_type`, and we have already cut that flow out with the previous condition. Let me know if I'm not seeing something.